src/models/instance.ts

```diff
@@ -103,6 +103,18 @@ export default class Instance {
         return new Instance({ ...this.data, description });
     }
 
+    public encryptPassword(encryptionKey: string) {
+        const cryptr = new Cryptr(encryptionKey);
+        const encrypted = cryptr.encrypt(this.data.password);
+        return new Instance({ ...this.data, password: encrypted });
+    }
+
+    public decryptPassword(encryptionKey: string) {
+        const cryptr = new Cryptr(encryptionKey);
+        const decrypted = cryptr.decrypt(this.data.password);
```

Comment:
When I try to edit an existing instance I get: `TypeError: invalid iv length 1`. That's normal, it tried to decrypt a plain password. Can you add a note in the section "notes for the tester" that old instances cannot be edited (should be deleted, I guess)?

Reply:
sure!